COPYING.md

```diff
@@ -225,52 +225,6 @@
 
     See COPYING file for the full LGPL text.
 
-## xz-minidec
-
-    XZ decompressor
-
-    Authors: Lasse Collin <lasse.collin@tukaani.org>
-            Igor Pavlov <http://7-zip.org/>
-
-    This file has been put into the public domain.
-    You can do whatever you want with this file.
-
-## ColumnResizer
-
-    Copyright (c) 2011-2016 Aurélien Gâteau and contributors.
-
-    All rights reserved.
-
-    Redistribution and use in source and binary forms, with or without
-    modification, are permitted (subject to the limitations in the
-    disclaimer below) provided that the following conditions are met:
-
-    * Redistributions of source code must retain the above copyright
-    notice, this list of conditions and the following disclaimer.
-
-    * Redistributions in binary form must reproduce the above copyright
-    notice, this list of conditions and the following disclaimer in the
-    documentation and/or other materials provided with the
-    distribution.
-
-    * The name of the contributors may not be used to endorse or
-    promote products derived from this software without specific prior
-    written permission.
-
-    NO EXPRESS OR IMPLIED LICENSES TO ANY PARTY'S PATENT RIGHTS ARE
-    GRANTED BY THIS LICENSE.  THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT
-    HOLDERS AND CONTRIBUTORS "AS IS" AND ANY EXPRESS OR IMPLIED
-    WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED WARRANTIES OF
-    MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
-    DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT OWNER OR CONTRIBUTORS BE
-    LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR
-    CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF
-    SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR
-    BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY,
-    WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE
-    OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN
-    IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
-
 ## launcher (`libraries/launcher`)
 
      PolyMC - Minecraft Launcher
```

Comment:
should this still me PolyMc?

Reply:
since it's all polymc code, yes

Reply:
thought this would be similar to the mention of multimc as it would branch of now